### PR TITLE
AG-1232 fixed nominations ordering bug, added tests

### PR DESCRIPTION
**Pull request by sagely1**

### CODE REVIEW

File: src/app/testing/gene-mocks.ts [hunk 1]
```diff
@@ -17,6 +17,23 @@ export const geneMock1: Gene = {
   is_any_protein_changed_in_ad_brain: true,
   protein_brain_change_studied: true,
   target_nominations: [
+    {
```
Comment (sagely1) [inline src/app/testing/gene-mocks.ts:20]: Added another target nomination to test.  Intentionally chose this one to be out of alphanumeric order.

File: src/app/testing/team-mocks.ts [hunk 2]
```diff
@@ -28,10 +28,10 @@ export const mockTeam2: Team = {
 
 export const mockTeam3: Team = {
   team: 'Emory-Sage-SGC',
-  team_full: 'Emory University',
+  team_full: 'Emory University - Sage Bionetworks - Structural Genomics Consortium',
   program: 'TREAT-AD',
   description:
-  "Emory University - Sage Bionetworks - Structural Genomics Consortium",
```
Comment (sagely1) [inline src/app/testing/team-mocks.ts:34]: incorrect mock data used

File: src/app/features/genes/components/gene-nominations/gene-nominations.component.ts [hunk 3]
```diff
@@ -36,35 +36,40 @@ export class GeneNominationsComponent {
...
+      //secondary sort on initial nomination year (descending)
+      return b.initial_nomination - a.initial_nomination;
+    });
   }
 
   getFullDisplayName(nomination: TargetNomination): string {
-    const team: Team = nomination.team_data as Team;
+    const team = nomination.team_data;
+    if (!team)
+      return '';
```
Comment (sagely1) [inline src/app/features/genes/components/gene-nominations/gene-nominations.component.ts:71]: this check was important to finding the nominations ordering bug

**Review by hallieswan — APPROVED** [review]
Looks good to me!

**Pull request merged**